modules/lymphgen/1.0/config/default.yaml

```diff
@@ -29,11 +64,16 @@ lcr-modules:
                 bcl6truevalues: "__UPDATE__"  # Can use a list
                 bcl6falsevalues: "__UPDATE__"
             lymphgen_input:
-                seq_type: "genome"  # Availible options: exome/genome (they do the same thing) and targeted (for panels)
+                seq_type: "__UPDATE__"  # Availible options: exome/genome (they do the same thing) and targeted (for panels)
                 use_log_ratio: "__UPDATE__"
```

Comment:
Maybe it makes sense to set the `use_log_ratio` value to `TRUE` as the default, since the CNV modules all output seg files with log ratio values. With this new set up I'm guessing it would be rare for this to be false.